### PR TITLE
Not extend Junit Assert classes in Tests. #1316

### PR DESCRIPTION
**Pull request by smadasu**
I have cleaned up all the Junit tests extending from Assert class. Please review

### CODE REVIEW

File: src/test/java/redis/clients/jedis/tests/ProtocolTest.java [hunk 1]
```diff
@@ -110,7 +115,10 @@ public void multiBulkReply() {
     expected.add(SafeEncoder.encode("Hello"));
     expected.add(SafeEncoder.encode("World"));
 
-    assertEquals(expected, response);
```
Comment (HeartSaVioR) [inline src/test/java/redis/clients/jedis/tests/ProtocolTest.java]: We can apply assertByteArrayListEquals this case, right?

File: src/test/java/redis/clients/jedis/tests/ProtocolTest.java [hunk 2]
```diff
@@ -110,7 +115,10 @@ public void multiBulkReply() {
...
+    assertEquals(expected.size(), response.size());
```
Comment (HeartSaVioR) [inline src/test/java/redis/clients/jedis/tests/ProtocolTest.java]: We can apply assertByteArrayListEquals this case, right?

**Comment by HeartSaVioR** [comment]
Thanks for the work. LGTM. Extending Assert was old fashion and this pull request corrects it.

@marcosnils Please review and merge. Thanks!

File: src/test/java/redis/clients/jedis/tests/ProtocolTest.java (hunk 2 above)
Comment (smadasu) [inline src/test/java/redis/clients/jedis/tests/ProtocolTest.java]: Yes. Thats right.

**Comment by marcosnils** [comment]
:+1: LGTM!. thx for the contribution @smadasu

**Comment by marcosnils** [comment]
Applied to 2.8 and 2.9 respectively. Thx again @smadasu

**Comment by smadasu** [comment]
Fantastic! Glad I could help :)

**Pull request merged**